compiler/src/dotty/tools/dotc/reporting/diagnostic/messages.scala

```diff
@@ -901,4 +900,23 @@ object messages {
     val msg = hl"trying to define package with same name as `$existing`"
     val explanation = ""
   }
+
+  case class ExistentialTypesNoLongerSupported()(implicit ctx: Context) extends Message(34) {
+    val kind = "Syntax"
+    val msg = "Existential types are no longer supported - use a wildcard or dependent type instead"
+    val explanation =
+      hl"""| The use of existential types is no longer supported and a wildcard or dependent type should be used instead
```

Comment:
This row is too long, max should be 80 characters, split it so that it is shorter than 80 (preferably even shorter).